configs/example/gpufs/system/system.py

```diff
@@ -231,6 +231,22 @@ def makeGpuFSSystem(args):
         clock=args.ruby_clock, voltage_domain=system.voltage_domain
     )
 
+    # If we are using KVM cpu, enable AVX. AVX is used in some ROCm libraries
+    # such as rocBLAS which is used in higher level libraries like PyTorch.
+    use_avx = False
+    if ObjectList.is_kvm_cpu(TestCPUClass):
+        # AVX also requires CR4.osxsave to be 1. These must be set together
+        # of KVM will error out.
+        system.workload.enable_osxsave = 1
+        use_avx = True
+
+    avx_extended_state = [0x00000007, 0x00000340, 0x00000000, 0x00000340,
```

Comment:
Is it possible to add a comment explaining what these numbers mean?

Reply:
Yes I can do that-  these are related to how much storage the CPU tells the CPU has for xsave.